pages/_app.tsx

```diff
@@ -4,7 +4,7 @@ import { ApolloClient } from '@apollo/client';
 import { ApolloProvider } from '@apollo/client';
 import { withShowcaseClient } from '@bcrumbs.net/bc-api';
 import '../public/assets/css/flaticon.css';
-import 'bootstrap-4-grid/css/grid.min.css';
+// import 'bootstrap-4-grid/css/grid.min.css';
```

Comment:
Why you have removed bootstrap, is it by purpose

Reply:
Yes, it gets me in problem sometimes